jussi/middlewares/jussi.py

```diff
@@ -52,6 +52,8 @@
         response.headers['x-jussi-response-time'] = str(now - request.get('timing', now))
 
         if request.method == 'POST':
```

Comment:
I think the solution here should be `if request.method == 'POST' and isinstance(request.json, dict)`
I say that because batch requests will alway be a list, and therefore not have the 'urn' attribute. I recognize that we aren't collecting metrics for batch requests, but we should really be using statsd for that anyways, as our current logging only permits us to sample 1% of requests anyways.